src/lib/components/schedule/schedules-interval-view.svelte

```diff
@@ -107,7 +101,7 @@
   <p>
     {translate('schedules', 'offset-description')}
```

Comment:
Are the different rounded borders here expected for the button vs. input? Also, looks like there's an extra border between the input and button now.

| Before | After |
|--|--|
|![Screenshot 2023-08-18 at 10 53 40 AM](https://github.com/temporalio/ui/assets/15069288/f6e08ea4-b298-4f71-a89b-3b32d50ce8c7)|![Screenshot 2023-08-18 at 10 15 06 AM](https://github.com/temporalio/ui/assets/15069288/387ff28b-fd8e-44f0-86b4-1cdc1fe5a9cb)|

Reply:
Hm yeah, maybe input should be 8px border-radius now as well

Reply:
I think I'm going to make a separate component, or maybe extend our `<Input />` component to support an adjacent select in a separate PR.